src/contour/helper.cpp

```diff
@@ -264,14 +264,19 @@ void spawnNewTerminal(string const& _programPath,
 {
     auto const wd = [&]() -> QString {
         auto const url = QUrl(QString::fromUtf8(_cwdUrl.c_str()));
+#if defined(__APPLE__)
```

Comment:
I don't understand this change. Why is this needed?

Reply:
because magically `url.host()` is always empty